src/main/java/duke/command/DeadlineCommand.java

```diff
@@ -0,0 +1,81 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.initials.Deadline;
+import duke.storage.Storage;
+import duke.tasklist.TaskList;
+import duke.ui.Ui;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class DeadlineCommand extends Command {
+
+    public static final SimpleDateFormat DATE_FORMAT = new SimpleDateFormat("dd/MM/yyyy HHmm");
+
+    /**
+     * Creates new deadline task using a TaskList, Ui and Storage, it will then be added into the taskArrayList that
+     * was loaded into the TaskList as param.
+     * @param tasks the TaskList to be used
+     * @param ui the Ui to be used
+     * @param storage the Storage to be used
+     * @throws DukeException
+     * @throws ParseException if the date is not able to be parsed
+     */
+    public void execute(TaskList tasks, Ui ui, Storage storage) throws DukeException, ParseException {
+
+        int end2 = ui.getRemainingWords().indexOf('/');
```

Comment:
haha because i already used the end variable in the previous method, so i initially used a weird lingo like "endies" but felt like I should use legit english and that's why i changed to end2